.test.env.dist

```diff
@@ -0,0 +1,15 @@
+# This file is a "template" of which env vars need to be defined for your application
+# Copy this file to .env file for development, create environment variables when deploying to production
+# https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
+
+APP_ENV=test
+APP_SECRET=33fc24e575b5b402d5aec5ffd335539b
+
+KERNEL_CLASS=App\Kernel
+
+# mongo test
+TEST_PAYUM_MONGO_DATABASE=${PAYUM_MONGO_DATABASE}_test
+TEST_PAYUM_MONGO_URI=mongodb://mongo/${TEST_PAYUM_MONGO_DATABASE}
```

Comment:
Add mongodb://localhost/${TEST_PAYUM_MONGO_DATABASE} since mongodb service is on the same host in travis.